man/CMakeLists.txt

```diff
@@ -1,43 +1,60 @@
-install(FILES
-	ceph-osd.8
-	ceph-mds.8
-	ceph-mon.8
-	ceph-syn.8
-	crushtool.8
-	osdmaptool.8
-	monmaptool.8
-	ceph-conf.8
-	ceph-run.8
-	ceph.8
-	mount.ceph.8
-	ceph-create-keys.8
-	radosgw.8
-	radosgw-admin.8
-	ceph-authtool.8
-	rados.8
-	librados-config.8
-	ceph-clsinfo.8
-	ceph-debugpack.8
-	cephfs.8
-	ceph-dencoder.8
-	ceph-rest-api.8
-	ceph-post-file.8
-  DESTINATION ${CEPH_MAN_DIR}/man8)
+set(manpages_srcs
```

Comment:
> not sure we will have a "make dist" for cmake

that's my understanding as well. i don't think we want to involve cpack, either

> have a CMakeLists.txt in into "/doc/man/8", and collect the man pages there

i would prefer it this way too. for one, it would mean that the entries in `manpages_srcs` would be relative to their CMAKE_CURRENT_SOURCE_DIR, rather than having to add the `${CMAKE_SOURCE_DIR}/doc/man/` prefix. it also solves the install to "man8" piece because each CMakeLists.txt could do its own install(). but i'm fine with putting this off until we need it